rekey.c

```diff
@@ -700,15 +700,15 @@ void rekey_verify_peer(struct candidate *peer) {
  * lifecycle
  */
 
-void rekey_init(service_context context, struct mesh_node *config) {
+void rekey_init(struct mesh_node *config, struct evl_ops *event_loop_ops) {
   if (cfg) {
     return;
   }
 
-  assert(context);
   assert(config);
+  assert(event_loop_ops);
 
-  ctx = context;
+  evl = event_loop_ops;
```

Comment:
oh so that is what evl stands for :) I was thinking "these operations are sooo evil"